<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QMainWindow" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="usersTbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>641</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addUserBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:green; color:white;font-size:18px;</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateUserBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>90</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:orange; color:white;font-size:18px;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteUserBtn">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:red; color:white;font-size:18px;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWelcome_Page">
    <property name="title">
     <string>Welcome Page</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHey">
    <property name="title">
     <string>Hey</string>
    </property>
   </widget>
   <addaction name="menuWelcome_Page"/>
   <addaction name="menuHey"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
